Main/Source/iconf.cpp

```diff
@@ -174,6 +180,10 @@ truthoption ivanconfig::HideWeirdHitAnimationsThatLookLikeMiss("HideWeirdHitAnim
                                           "Hide hit animations that look like miss",
                                           "",
                                           true);
+truthoption ivanconfig::UseLightEmiterBasedOnVolume("UseLightEmiterBasedOnVolume",
+                                          "Small crystal rocks etc. will emit less light.",
```

Comment:
No fullstop. :) Maybe "Small light sources emit less light" to make it less specific?

Reply:
it is more accurate indeed!